import-service/handlers/importFileParser.ts

```diff
@@ -12,6 +12,7 @@ export async function importFileParser(
     console.log('event', JSON.stringify(event, null, 2));
     try {
         const s3 = new AWS.S3({region: 'us-east-1'});
+        const sqs = new AWS.SQS({region: 'us-east-1'});
```

Comment:
I'd recommend moving `us-east-1` to constants or .env file to reduce duplications

Reply:
Done